index.php

```diff
@@ -409,9 +482,96 @@ function initMap() {
+  <script>
+      
+$(document).ready( function () {  
+
+        const userId=<?php echo($_SESSION['user_id']); ?>;
+        
+        //Lock Input Field By Default as Modal is Opened    
+        $("#my-profile").click(function () {
+          $("input").prop('disabled', true);
+          $("input").css('background-color','antiquewhite');
+          // $(this).prev().css("background-color","antiquewhite");
+          $("#apply-changes-btn").css({'background-color':'#6c757d','font-weight':'500','cursor': 'not-allowed'});
+          
+          //Fetch from Database using ajax
+          
+        /*   $.ajax({
+                  url: "fetch-user.php",
+                  method:"POST",
+                  data:{userId:userId},
+                  success:function(response)
+                  {
+                    //Encode response into array
+                    arr=JSON.parse(response);
+                    
+                    //Insert the fetched data into modal input fields
+                    $("#full-name").val(arr[0]["name"]);
+                    $("#email").val(arr[0]["email"]);
+                    $("#organisation").val(arr[0]["organisation"]);
+                    $("#state").val(arr[0]["state"]);
+                  }
+                });
+ */
+           
+          });
+
+        //Unlock respective input field as edit btn is clicked  
+        $(".edit-btn").on('click', function(){
+          console.log("edit clciked");
+          $(this).prev().prop("disabled",false);
+          $(this).prev().css("background-color","aliceblue");
+          $(this).prev().focus();
+        });
+
+        $(".modal .modal-body input").change( function(){
+            $("#apply-changes-btn").prop("disabled",false);
+            $("#apply-changes-btn").css({'background-color':'#28a745','font-weight':'600','cursor': 'pointer'});
+        });
+
+        $(function () {
+          $('[data-toggle="tooltip"]').tooltip()
+        })
+
+      $("#apply-changes-btn").click( function(){
+        console.log("Apply btn clicked");
+        let  name=$("#full-name").val();
+        let  organisation=$("#organisation").val();
+        let  state=$("#state").val();
+         
+        /* $.ajax({
+                url: "edit-user.php",
+                method:"POST",
+                data:{userId:userId,name:name,organisation:organisation,state:state},
+                success:function(response)
+                {
+                  alert(response);
+                  $("input").prop('disabled', true);
+                  $("input").css('background-color','antiquewhite');
+                  $("#apply-changes-btn").prop("disabled",false);
+                  $("#apply-changes-btn").css({'background-color':'#6c757d','font-weight':'500','cursor': 'not-allowed'});
+                  $(".nav-username h2,.menu-username h2").html('Welcome '+name+'!');
+                  $(".modal-title").html(name+"'s Profile");
+                }
+              }); */
+
```

Comment:
Code to update the user profile, if clicked on 'Save Changes' Button. This has been commented as well